README.md

```diff
@@ -58,6 +62,28 @@ If you cannot control how your program is run, you can load the `newrelic` modul
     /* ... the rest of your program ... */
 ```
 
+## ES Modules
+
+If your application is written with `import` and `export` statements and is not transpiled to javascript, you are using [ES Modules](https://nodejs.org/api/esm.html#modules-ecmascript-modules) and must bootstrap the agent in a different way. 
```

Comment:
"and is not transpiled" is a bit of a funny way of saying "and it's not typescript", as if import and export weren't javascript too.

How about, "if your application is using import and export modules in javascript"? Or should we explicitly say "and it's not typescript"? Not sure what other things could use import and export.

Reply:
you can be using babel to transpile native javascript, not sure why you would sitll want to but you can. but i'll just be more concise like "if your application is using import and export modules in javascript"